service.cronxbmc/resources/lib/__init__.py

```diff
@@ -0,0 +1,5 @@
+# For relative imports to work from Python 3.6
+import os
+import sys
+
+sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
Please drop, no need to dummy init files, also please avoid syspath append and just use relative module import

Reply:
I added this to make it possible to import cron from another addon. without this I get:

```
                                                   Error Type: <class 'ModuleNotFoundError'>
                                                   Error Contents: No module named 'resources.lib.croniter'
                                                   Traceback (most recent call last):
                                                     File "/home/termo/.kodi/addons/plugin.video.drnu/default.py", line 2, in <module>
                                                       from resources.lib import addon
                                                     File "/home/termo/.kodi/addons/plugin.video.drnu/resources/lib/addon.py", line 28, in <module>
                                                       from cron import CronManager, CronJob
                                                     File "/home/termo/.kodi/addons/service.cronxbmc/resources/lib/cron.py", line 6, in <module>
                                                       from resources.lib.croniter import croniter
                                                   ModuleNotFoundError: No module named 'resources.lib.croniter'
                                                   -->End of Python script error report<--

```